src/sfrpg.js

```diff
@@ -665,6 +665,10 @@ function setupHandlebars() {
         const formattedValue = moneyFormatter.format(value);
         return formattedValue;
     });
+
+    Handlebars.registerHelper('json', function(context) {
```

Comment:
Also how does this differ from the built-in `{{log}}` function?

Reply:
> Also how does this differ from the built-in `{{log}}` function?

funny didn't even know this `log` thing exists. I would say the main difference is this one prints the object directly onto the place it's put on as a string. Which made it possible for me to print the json from `section.classes` directly into the data attribute. 
But as this will now be changed this will not be needed anymore. Please decide if this should stay or not.